### PR TITLE
Relay endpoints

### PR DESCRIPTION
**Pull request by jaind1046**
Added steps for relay endpoint features

### CODE REVIEW

**Comment by jaind1046** [comment]
@NadHodeGW Relay endpoints tests for review please

File: specs/features/transaction-log-view.feature [hunk 1]
```diff
@@ -1,18 +1,245 @@
...
+		When I set file name to a valid value
+		When I selected 2 file outcomes
+		When I clicked go
+		Then I am on page 1
+
+  	############################################
+	# Pagination Scenarios
+	############################################
+
+	@TC_Transaction_Logs_Navigation
```
Comment (NadHodeGW) [inline specs/features/transaction-log-view.feature:227]: Invalid scenario syntax, do not have more than 3 when, then or given lines in a scenario; Also, no when step after a then step; please change

**Review by NadHodeGW — CHANGES_REQUESTED** [review]
1- See comment in transaction-log-view.feature and update
2- new added scenarios in transaction-log-view.feature do not provide any value unless updated to to have an outcome, the Then step should result to a filtered log based on the set or unset filter

**Review by werzl — APPROVED** [review]

**Comment by jaind1046** [comment]
Moving to data driven approach. Merge not needed.Closing

**Pull request closed without merging**